src/cli/utils/exchange.go

```diff
@@ -128,30 +128,32 @@ func ValidateExchangeRestoreFlags(backupID string, opts ExchangeOpts) error {
 
 // IncludeExchangeRestoreDataSelectors builds the common data-selector
 // inclusions for exchange commands.
-func IncludeExchangeRestoreDataSelectors(
-	sel *selectors.ExchangeRestore,
-	opts ExchangeOpts,
-) {
+func IncludeExchangeRestoreDataSelectors(opts ExchangeOpts) *selectors.ExchangeRestore {
+	users := opts.Users
+	if len(users) == 0 {
+		users = selectors.Any()
+	}
+
+	sel := selectors.NewExchangeRestore(users)
+
 	lc, lcf := len(opts.Contact), len(opts.ContactFolder)
 	le, lef := len(opts.Email), len(opts.EmailFolder)
 	lev, lec := len(opts.Event), len(opts.EventCalendar)
 	// either scope the request to a set of users
 	if lc+lcf+le+lef+lev+lec == 0 {
-		if len(opts.Users) == 0 {
-			opts.Users = selectors.Any()
-		}
-
-		sel.Include(sel.Users(opts.Users))
+		sel.Include(sel.Users(users))
```

Comment:
is this still required? We're already setting users on line 137

Reply:
Removal will happen in one of the next PRs.